js/index.js

```diff
@@ -51,15 +51,22 @@ let middleImg = document.getElementById('middle-img')
 
 
 //navBar
-navBar[0].textContent = siteContent['nav'][`nav-item-1`]
-navBar[1].textContent = siteContent['nav'][`nav-item-2`]
-navBar[2].textContent = siteContent['nav'][`nav-item-3`]
-navBar[3].textContent = siteContent['nav'][`nav-item-4`]
-navBar[4].textContent = siteContent['nav'][`nav-item-5`]
-navBar[5].textContent = siteContent['nav'][`nav-item-6`]
+let count = 1;
+navBar.map(item => {
+  item.textContent = siteContent['nav'][`nav-item-${count}`]
```

Comment:
Good job refactoring some code that are repetitive and make use of loop to solve it! 🔥